modules/applications/examples/ui-builder/production/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Used for running a bonita env for production
+services:
+  bonita-ui-builder:
+    image: <my-uib-image-name>:<version>
+    container_name: bonita-ui-builder
+    environment:
+      BONITA_DEV_MODE: false
+      BONITA_API_URL: http://host.docker.internal:8080/bonita/API
+      APPSMITH_ENCRYPTION_PASSWORD: <encryption-password>
+      APPSMITH_ENCRYPTION_SALT: <encryption-salt>
+    ports:
+      - "8090:80"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    depends_on:
+      bonita:
```

Comment:
**question:** It's normal that is not the same name as `bonita-app`?

Reply:
my bad. Good catch.